src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -471,6 +576,35 @@ private ICredentialProvider GetCredentialProvider(CommandSettings command, strin
             provider.EnsureCredential(HostContext, command, serverUrl);
             return provider;
         }
+        private void AssertAdminAccess()
```

Comment:
we saw some inbetween iteration.

---
In reply to: [117774956](https://github.com/Microsoft/vsts-agent/pull/978#discussion_r117774956) [](ancestors = 117774956)